frontend/src/api/api.js

```diff
@@ -15,3 +15,45 @@ export async function registerUser(
 
     return response.data.message;
 }
+
+export async function loginUser(email, password) {
+    const response = await userInstance.post("/login", {
+        email,
+        password,
+    });
+
+    return response.data;
+}
+
+export async function logout() {
+    const token = localStorage.getItem("token");
+    const response = await userInstance.get("/logout", {
+        headers: {
+            Authorization: `Bearer ${token}`,
+        },
+    });
+    window.location.replace = "/login";
+
+    console.log(response.data);
```

Comment:
ayaw pag butang other logic dre. adto ni eadd ang redirection kung asa nagtawag og logout.
```suggestion
    return response;
```